app.py

```diff
@@ -0,0 +1,634 @@
+                                        "outline": False,
+                                    },
+                                ),
+                            ]
+                        ),
+                        className="border-0 bg-transparent",
+                    ),
+                    id="collapse",
+                    is_open=False,
+                ),
+            ]
+        )
+    ],
+    className="border-0 bg-transparent",
+)
+
+availability_card = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H2(
+                    children="Employee Availability", style={"font-family": ff}
+                ),
+                html.Div(
+                    id="initial-sched",
+                    style={
+                        "marginBottom": "5px",
+                        "outline": False,
+                    },
+                ),
+            ]
+        )
+    ]
+)
+
+ready_style = {
+    "max-width": "50%",
+    "marginBottom": "5px",
+    "outline": False,
+}
+pending_style = {
+    "max-width": "50%",
+    "marginBottom": "5px",
+    "outline": False,
+    "background-color": "red",
+}
+solve_card = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                dbc.Button("Solve CQM", id="btn_solve_cqm", style=ready_style),
+                html.Div(id="trigger", children=0, style=dict(display="none")),
+            ]
+        )
+    ],
+    className="border-0 bg-transparent",
+)
+
+schedule_card = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                html.H2(
+                    children="Employee Schedule", style={"font-family": ff}
+                ),
+                html.Div(
+                    id="built-sched",
+                    style={
+                        "marginBottom": "5px",
+                        "outline": False,
+                        "textAlign": "center",
+                    },
+                ),
+            ]
+        )
+    ]
+)
+
+legend_card = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                dbc.Row(
+                    dbc.FormText(
+                        "Employee unavailable",
+                        style={
+                            "color": col,
+                            "backgroundColor": "#FF7006",
+                            "padding": 10,
+                        },
+                    )
+                ),
+                dbc.Row(
+                    dbc.FormText(
+                        "Preferred shift",
+                        style={
+                            "color": col,
+                            "backgroundColor": "#008c82",
+                            "padding": 10,
+                        },
+                    )
+                ),
+            ]
+        )
+    ]
+)
+
+shift_legend_card = dbc.Card(
+    [
+        dbc.CardBody(
+            [
+                dbc.Row(
+                    dbc.FormText(
+                        "Employee shift scheduled",
```

Comment:
This Employee Schedule screen needs additional legend items: X and P (are only described in the previous Availability screen) and "-" (dash) is not described anywhere